code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -47,3 +47,34 @@
 	embedding = list(embed_chance=100, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
 	wound_falloff_tile = -5
 	shrapnel_type = /obj/item/ammo_casing/harpoon
+
+// Rebar (Rebar Crossbow)
+/obj/projectile/bullet/rebar
+	name = "rebar"
+	icon_state = "rebar"
+	damage = 40
+	stamina = 20
+	speed = 0.4
+	dismemberment = 1 //because a 1 in 100 chance to just blow someones arm off is enough to be cool but also not enough to be reliable
+	armour_penetration = 12
+	wound_bonus = 0
```

Comment:
I would seriously consider giving this a negative value. At +40 wounding power before armor reduction, this is almost always a catastrophic wound even against armored targets. Something close to -20 or 30 would be appropriate.

Reply:
> I would seriously consider giving this a negative value. At +40 wounding power before armor reduction, this is almost always a catastrophic wound even against armored targets. Something close to -20 or 30 would be appropriate.

Okay! will do. I only had basic humans to test it on runtime so I wasnt sure how nasty the wounds were,